frontend/src/lib/error/index.ts

```diff
@@ -83,7 +97,15 @@ function setupGlobalErrorHandlers(error: Writable<App.Error | null>): void {
       return;
     }
 
+    if (!showToUser) return;
+
     const updateDetected = await updated.check();
     error.set({message: message ?? `We're not sure what happened.`, traceId, handler, updateDetected});
   };
+
+  function errorIsFromExtension(error: Error): boolean {
+    return !!error.stack &&
```

Comment:
so if there's no stack trace then it's not from an extension? I thought you mentioned that sometimes if it is null then it's likely from an extension?

Reply:
What I have observed are:
- In the case of LastPast, there is simply no error object on the event
- In the case of Tampermonkey, there's an error object and it has a stack trace

So, so far there's no evidence that an error-event from an extension would have an error object without a stack trace.